### PR TITLE
 [freeze][3005] Fix branch tests

### PR DESCRIPTION
**Pull request by Ch3LL**

### CODE REVIEW

**Comment by Ch3LL** [comment]
re-run full all

**Comment by Ch3LL** [comment]
re-run full all

**Comment by Ch3LL** [comment]
re-run full all

**Comment by Ch3LL** [comment]
re-run full all

**Comment by Ch3LL** [comment]
re-run full all

**Comment by Ch3LL** [comment]
re-run full all

File: tests/pytests/scenarios/setup/test_install.py [hunk 1]
```diff
@@ -28,6 +28,8 @@ def use_static_requirements_ids(value):
 
 @pytest.fixture(params=[True, False], ids=use_static_requirements_ids)
 def use_static_requirements(request):
+    if not request.param and salt.utils.platform.is_windows():
+        pytest.skip("Windows installs with statis requirements only")
```
Comment (s0undt3ch) [inline tests/pytests/scenarios/setup/test_install.py:32]:
```suggestion
        pytest.skip("Windows installs with static requirements only")
```

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

**Comment by Ch3LL** [comment]
re-run full all

**Review by garethgreenaway — APPROVED** [review]

**Review by dmurphy18 — APPROVED** [review]

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**